kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/web/EcosystemWebView.java

```diff
@@ -44,7 +46,9 @@ public EcosystemWebView(Context context, AttributeSet attrs, int defStyle) {
 
         nativeApi = new EcosystemNativeApi();
         addJavascriptInterface(nativeApi, JS_INTERFACE_OBJECT_NAME);
-        setWebContentsDebuggingEnabled(DEBUG);
+        if (VERSION.SDK_INT >= VERSION_CODES.KITKAT) {
+            setWebContentsDebuggingEnabled(DEBUG);
```

Comment:
we will need to remove this if we run on production environment right?

Reply:
it's using  "DEBUG" from the build itself so it won't be on prod.
but we can remove that for release, right now I guess Lior uses it.